strapi/strapi
# fix: createdBy missing when you create a variant of a document in a new locale

### What does it do?

Fixes this issue:

https://github.com/strapi/strapi/issues/21270

Just on collection Type because in Single type the ternary option was already in place

### How to test it?

Follow the steps above:
- enable i18n in a collection and add a new locale in the settings
- create a document on the default locale in the collection and save, check the string "Created n days ago by ..." in the "More" dropdown in the top right of the edit view
- change the locale and update the document, check the string "Created n days ago by ..." in the "More" dropdown in the top right of the edit view
